src/backend/be_utils.js

```diff
@@ -0,0 +1,35 @@
+const BD_TRANSACAO = "";
+
+const json_test = {
+        "_id": "1",
+        "data": "Thu Jun 21 2023 09:54:51 GMT-0300",
+        "tipo": "cashback",
```

Comment:
is "cashback" when you pay using your credits? Or is this the amount you got as cashback?
if is the latter, than every "cashback" is paired with a "payment" right? And how would you represent a payment using credits ?

Reply:
Yes, 'cashback' refers to the amount that you get when you pay using a different type of payment. Now the types are: 
- 'pagamento': when paid **not** using your credits
- 'pagamento_saldo': when using your credits
- 'cashback': amount received when a payment of type 'pagamento' is made

At the first moment, cashback and payments will not be related.